tools/isobuild/builder.js

```diff
@@ -393,7 +395,10 @@ Previous builder: ${previousBuilder.outputPath}, this builder: ${outputPath}`
   // A version of copyDirectory that works better for copying node_modules
   // directories when symlinks are involved.
   copyNodeModulesDirectory(options) {
-    assert.strictEqual(files.pathBasename(options.from), "node_modules");
+    // Although the options.from directory should probably be a
+    // node_modules directory, the only essential precondition here is
+    // that the destination directory is a node_modules directory.
+    // assert.strictEqual(files.pathBasename(options.from), "node_modules");
```

Comment:
Just flagging this commented out line  Not sure if intended to leave as a historical artifact.

Reply:
Yeah, I decided it to leave it there because it makes the comment more concrete. Also, we might want to reinstate the assertion at some point in the future, once we're confident there shouldn't be any more `node_modules1` directories hanging around.